src/reference-manual/types.Rmd

```diff
@@ -1524,6 +1611,7 @@ assignable to the variable being declared.  For example,
 
 ```stan
 matrix[3, 2] a = 0.5 * (b + c);
+complex z = 2 + 3i;
```

Comment:
I think this should be split out into a second example or at least mentioned in the prose that follows

Reply:
split it out.